core/src/main/java/com/linecorp/armeria/client/endpoint/PropertiesEndpointGroup.java

```diff
@@ -36,9 +40,11 @@
  * A {@link Properties} backed {@link EndpointGroup}. The list of {@link Endpoint}s are loaded from the
  * {@link Properties}.
  */
-public final class PropertiesEndpointGroup implements EndpointGroup {
+public final class PropertiesEndpointGroup extends DynamicEndpointGroup {
+    private static final PropertiesFileWatcherRegistry registry = new PropertiesFileWatcherRegistry();
 
-    // TODO(ide) Reload the endpoint list if the file is updated.
+    @Nullable
+    private Runnable closeCallback;
```

Comment:
- This field has to be moved after the static methods.
- This field could be `final` if it is set with `null` when unused.